yarn-project/prover-client/src/prover-agent/prover-agent.ts

```diff
@@ -49,11 +58,23 @@ export class ProverAgent {
       throw new Error('Agent is already running');
     }
 
+    let lastPrint = process.hrtime.bigint();
+
     this.runningPromise = new RunningPromise(async () => {
       for (const jobId of this.inFlightPromises.keys()) {
         await jobSource.heartbeat(jobId);
       }
 
+      const now = process.hrtime.bigint();
+
+      if (now - lastPrint >= PRINT_THRESHOLD_NS) {
+        const jobs = Array.from(this.inFlightPromises.values())
+          .map(job => `id=${job.id},type=${job.type}`)
```

Comment:
For the type, don't you need to do something like ${ProvingRequestType[job.type]}?

Reply:
great catch! fixed